script/server

```diff
@@ -47,15 +47,20 @@ startup()
     if have systemctl; then
         systemctl is-active rsyslog || sudo systemctl start rsyslog || die 'Failed to start rsyslog!'
         systemctl is-active dbus || sudo systemctl start dbus || die 'Failed to start dbus!'
-        systemctl is-active avahi-daemon || sudo systemctl start avahi-daemon || die 'Failed to start avahi!'
+        sudo service mdns status || sudo service mdns start || echo "service mdns is not available!"
+        if [[ ${OTBR_MDNS} == "avahi" ]]; then
```

Comment:
You need the similar change in this file, like
```
OTBR_MDNS="${OTBR_MDNS:-mDNSResponder}"
```

Reply:
Thank you! I added it and pushed a new version.